Tests/SonarQube.Bootstrapper.Tests/BootstrapperExeTests.cs

```diff
@@ -57,12 +57,14 @@ public void Exe_PreProc_UpdateFails()
                 mockUpdater.TryUpdateReturnValue = false;
 
                 // Act
-                TestLogger logger = CheckExecutionFails(mockUpdater, "/d:sonar.host.url=http://host:9000");
+                TestLogger logger = CheckExecutionFails(mockUpdater, "/d:sonar.host.url=http://host:9000", "begin");
```

Comment:
I've updated the bootstrapper tests to so that all of the non-compat tests supply the expected verbs and check there are no warnings.
The compat test does not supply the verbs and checks the warnings appear.